src/app/objectdetails/objectdetails.component.html

```diff
@@ -5,6 +5,11 @@
 <div *ngIf="_mediaObject as obj" style="padding:10px;">
     <app-objectviewer #objectviewerComponent [mediaobject]="obj"></app-objectviewer>
 
+    <mat-button-toggle-group name="orderType" aria-label="Order By" [value]="orderTypeStr" (change)="onOrderTypeChange($event.value)" >
+        <mat-button-toggle value="score">Score</mat-button-toggle>
```

Comment:
I don't like this huge UI elements for this particular feature and request the usage of a [slide toggle](https://material.angular.io/components/slide-toggle/examples)

Reply:
My reasoning was that there might be additional different sorting options in the future. Slide toggles indicate to me that something is on or off - here we choose between two equally valid options.